tools/compile_tf_graph.py

```diff
@@ -741,9 +911,7 @@ def main(argv):
       from TFUtil import CollectionKeys
       with open(args.output_file_state_vars_list, "w") as f:
         for param in tf.get_collection(CollectionKeys.STATE_VARS):
-          assert param.name[-2:] == ":0"
-          f.write("%s\n" % param.name[:-2])
-
-
+          f.write("%s\n" % param.name)
```

Comment:
Why this change?

Reply:
stat vars created locally does not have :0 at the end

Reply:
What is the type of `param`? `tf.Tensor.name` always has this. It might be a `tf.Variable` then. But then better check thet, i.e. add an `assert isinstance(param, tf.Variable)`.